README.md

```diff
@@ -11,7 +11,7 @@ docker create \
 	-v </path/to/data>:/data \
 	-v </path/to/path>:/music:ro \
 	-p 3030:80 \
-	thetarkus/funkwhale
+	funkwhale/all-in-one
```

Comment:
There is possibly a small issue here, since we don't have any `latest` tag here. In the meantime, I'd suggest providing an explicit tag, like that:

```
export FUNKWHALE_VERSION=0.18.1
docker run
...
    funkwhale/all-in-one:$FUNKWHALE_VERSION
```